packages/framework/src/Migrations/Version20251124135249.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Override;
+use Shopsys\FrameworkBundle\Component\Translation\Translator;
+use Shopsys\FrameworkBundle\Model\Order\Status\OrderStatusTypeEnum;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+
+class Version20251124135249 extends AbstractMigration implements ContainerAwareInterface
+{
+    use MultidomainMigrationTrait;
+
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    #[Override]
+    public function up(Schema $schema): void
+    {
+        $nextId = (int)$this->sql('SELECT MAX(id) + 1 FROM order_statuses')->fetchOne();
+
+        $this->sql('INSERT INTO order_statuses (id, type) VALUES (:id, :type)', [
+            'id' => $nextId,
+            'type' => OrderStatusTypeEnum::TYPE_WITHDRAWN,
+        ]);
+
+        foreach ($this->getAllLocales() as $locale) {
+            $this->sql(
+                'INSERT INTO order_status_translations (translatable_id, name, locale) VALUES (:id, :name, :locale)',
+                [
+                    'id' => $nextId,
+                    'name' => t('Withdrawn', [], Translator::DEFAULT_TRANSLATION_DOMAIN, $locale),
+                    'locale' => $locale,
+                ],
+            );
+        }
+
+        $this->sql('SELECT setval(\'order_statuses_id_seq\', GREATEST(MAX(id), :nextId)) FROM order_statuses', ['nextId' => $nextId]);
```

Comment:
Wouldn't it be better to let PostgreSQL handle sequences automatically and just select the associated ID instead of forcing it and setval the sequence?

Reply:
Ah, of course, that is way better, thank you! 136322ecb5

Reply:
The only thing that bothers me - because of that already removed vintage "over limit" order status, the next sequence number is actually 6, so we now have a gap there. Should we care? 
<img width="226" height="201" alt="image" src="https://github.com/user-attachments/assets/eb7c2d41-0f80-489e-bd39-efdf18f7ce07" />